syscalls.c

```diff
@@ -920,17 +928,18 @@ addr_t syscalls_va2pa(void *ustack)
 
 	GETFROMSTACK(ustack, void *, va, 0);
 
-	return (pmap_resolve(proc_current()->process->pmapp, (void *)((ptr_t)va & ~0xfff)) & ~0xfff) + ((ptr_t)va & 0xfff);
+	return (pmap_resolve(proc_current()->process->pmapp, (void *)((ptr_t)va & ~0xfffU)) & ~0xfffU) + ((ptr_t)va & 0xfffU);
 }
 
 
 int syscalls_signalHandle(void *ustack)
 {
-	void *handler;
+	typedef void (*handler_func)(void);
```

Comment:
same as above - maybe we should move the type definition to some more sane place?